docs/design.md

```diff
@@ -104,3 +104,47 @@ several ways according to policy:
 
 Then like the usual LibOS pattern it will call the container's entry point as though the container
 was a process unto itself (rather than where it now sits like a library in the LibOS PAL process).
+
+## [Future Work] Local Attestation for Filesystem Decryption Key Provisioning
+
+![EnclaveCC_LA.png](images/EnclaveCC_LA.png)
+
+A local attestation (LA) feature is needed for authentication/authorization and secret provisioning
+between an Agent-enclave instance and a set of App-enclave instances resided in a same Pod. The
+agent holds a secret for filesystem decryption and the app wants to authenticate to the agent in
+order for the agent to release the filesystem decryption key (FDK). The app will prove its identity
```

Comment:
```suggestion
order for the agent to release the filesystem decryption key (FDK). The app-enclave will prove its identity
```

Reply:
Thank you so much for those commit suggestions.
It is very helpful to improve the readability. Thanks.